cg/cli/clean.py

```diff
@@ -274,11 +272,9 @@ def clean_flow_cells(context: CGConfig, dry_run: bool):
                 dry_run=dry_run,
             )
             clean_flow_cell_api.delete_flow_cell_directory()
-        except CleanFlowCellFailedError as error:
+        except (CleanFlowCellFailedError, FlowCellError) as error:
             LOG.error(repr(error))
-            exit_code = EXIT_FAIL
-    if exit_code:
-        click.Abort
+            continue
```

Comment:
Will this work? Seems as if the every error in delete_flow_cell_directory is wrapped into a CleanFlowCellFailedError.

Reply:
Or right! That error is raised in the instantiation?